prov/sm2/src/sm2_coordination.h

```diff
@@ -48,7 +48,7 @@
 
 #include <rdma/providers/fi_prov.h>
 
-#define SM2_INJECT_SIZE	      4096
+#define SM2_INJECT_XFER_SIZE  4096
```

Comment:
Again- still confusing. I would rename to SM2_XFER_ENTRY_SIZE. Otherwise that looks like it's the actual transfer size